src/lib/expansion/expansion.md

```diff
@@ -98,3 +98,10 @@ panel can be expanded at a given time:
 
 </md-accordion>
 ```
+
+### Accessibility
+The expansion panel header has `role="button"`. The expansion panel header has attribute
+`aria-controls` with the expansion panel's id as value.
+
+The expansion panel headers are buttons. Users can use the keyboard to activate the expansion panel
+header to switch between expanded state and collapsed state.
```

Comment:
I'd add another sentence 
```markdown
Because the header acts as a button, additional interactive elements
should not be put inside of the header.
```